src/Microsoft.ML.Data/Transforms/HashTransform.cs

```diff
@@ -2,23 +2,30 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Runtime.CompilerServices;
-using System.Text;
+using Microsoft.ML.Core.Data;
 using Microsoft.ML.Runtime;
 using Microsoft.ML.Runtime.CommandLine;
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
 
-[assembly: LoadableClass(HashTransform.Summary, typeof(HashTransform), typeof(HashTransform.Arguments), typeof(SignatureDataTransform),
+[assembly: LoadableClass(HashTransform.Summary, typeof(IDataTransform), typeof(HashTransform), typeof(HashTransform.Arguments), typeof(SignatureDataTransform),
     "Hash Transform", "HashTransform", "Hash", DocName = "transform/HashTransform.md")]
 
-[assembly: LoadableClass(HashTransform.Summary, typeof(HashTransform), null, typeof(SignatureLoadDataTransform),
+[assembly: LoadableClass(HashTransform.Summary, typeof(IDataTransform), typeof(HashTransform), null, typeof(SignatureLoadDataTransform),
     "Hash Transform", HashTransform.LoaderSignature)]
 
+[assembly: LoadableClass(HashTransform.Summary, typeof(HashTransform), null, typeof(SignatureLoadModel),
+     "Hash Transform", HashTransform.LoaderSignature)]
+
+[assembly: LoadableClass(typeof(IRowMapper), typeof(HashTransform), null, typeof(SignatureLoadRowMapper),
+   "Hash Transform", HashTransform.LoaderSignature)]
+
 namespace Microsoft.ML.Runtime.Data
```

Comment:
>Microsoft.ML.Runtime.Data [](start = 10, length = 25)

let's get into habit of setting the correct namespace and naming.
1) `namespace Microsoft.ML.Transforms`
2) Estimators should be named the same as the `Legacy` ones as appropriate, with no Estimator suffix. In this case, it should be `HashConverter`.
3) Transformers should be named the same as estimators, with a `Transformer` suffix: `HashConverterTransformer`.